tests/plugins/image2/upcast.js

```diff
@@ -37,7 +37,7 @@
 				name: 'enterP',
 				data: '<p style="text-align:center">' +
 					'<img id="w1" src="_assets/foo.png" alt="foo" />' +
-				'</p>'
+					'</p>'
```

Comment:
Just hit that autoformatting keystrong, although that's correct indentation so I'm keeping that.

Reply:
Actually it's not correct, as nested `img` tag is on the same level as `p` now.

Reply:
You are right. It's not correct as html markup it would be for regular string, however original one wasn't correct either, as `'</p>'` shouldn't be on same level as `data` property.